test/fixtures/wpt/resources/webidl2/README.md

```diff
@@ -5,7 +5,7 @@
 
 ## Purpose
 
-This is a parser for the [WebIDL](http://dev.w3.org/2006/webapi/WebIDL/) language. If
+This is a parser for the [WebIDL](https://heycam.github.io/webidl/) language. If
```

Comment:
Even though this currently redirects, it's to someone's personal github account. I think we should probably keep the original URL in case the redirect is changed or removed in the future. Just changing the 'http' to 'https' would be enough here I think.